src/viur/toolkit/importer/importable.py

```diff
@@ -88,6 +88,7 @@ class Importable:
         "translate.ignore": None,  # Bones to be ignored in automatically generated translation
         "action": "list",  # Action to run, default is "list"
         "limit": 99,  # Amount of items to fetch per request (only for list-ables)
+        "debuglimit": None,  # Number of items to import (use for debugging only!)
```

Comment:
Can you add some more description about the behavior like "after this amount of entries we stop the import"

Reply:
```suggestion
        "debuglimit": None,  # After this amount of entries we stop the import (use it for debugging only!)
```